Gemfile

```diff
@@ -16,8 +16,6 @@ gem "react-rails"
 
 gem 'accountingjs-rails'
 gem "jquery-rails"
-gem "turbolinks"
```

Comment:
I'm sure we could debug the problems (but the breadcrumbs weren't the only place of run into this)...  Perhaps we file a nice to have issue to add it back in...